pkg/ecdsa/dkg/member.go

```diff
@@ -111,24 +126,185 @@ func (skgm *symmetricKeyGeneratingMember) MarkInactiveMembers(
 	filter.FlushInactiveMembers()
 }
 
+// initializeTssRoundOne returns a member to perform next protocol operations.
+func (skgm *symmetricKeyGeneratingMember) initializeTssRoundOne() *tssRoundOneMember {
+	tssPartyID, groupTssPartiesIDs := generateTssPartiesIDs(
+		skgm.id,
+		skgm.group.MemberIDs(),
+	)
+
+	tssParameters := tss.NewParameters(
+		tss.EC(),
+		tss.NewPeerContext(tss.SortPartyIDs(groupTssPartiesIDs)),
+		tssPartyID,
+		len(groupTssPartiesIDs),
+		skgm.group.DishonestThreshold(),
+	)
+
+	tssOutgoingMessagesChan := make(chan tss.Message, len(groupTssPartiesIDs))
+	tssResultChan := make(chan keygen.LocalPartySaveData)
+
+	tssParty := keygen.NewLocalParty(
+		tssParameters,
+		tssOutgoingMessagesChan,
+		tssResultChan,
+		*skgm.tssPreParams,
+	)
+
+	return &tssRoundOneMember{
+		symmetricKeyGeneratingMember: skgm,
+		tssParty:                     tssParty,
+		tssParameters:                tssParameters,
+		tssOutgoingMessagesChan:      tssOutgoingMessagesChan,
+		tssResultChan:                tssResultChan,
+	}
+}
+
+// tssRoundOneMember represents one member in a distributed key generating
+// group performing the first round of the TSS keygen.
+type tssRoundOneMember struct {
+	*symmetricKeyGeneratingMember
+
+	tssParty                tss.Party
+	tssParameters           *tss.Parameters
+	tssOutgoingMessagesChan <-chan tss.Message
+	tssResultChan           <-chan keygen.LocalPartySaveData
+}
+
+// initializeTssRoundTwo returns a member to perform next protocol operations.
+func (trom *tssRoundOneMember) initializeTssRoundTwo() *tssRoundTwoMember {
+	return &tssRoundTwoMember{
+		tssRoundOneMember: trom,
+	}
+}
+
+// tssRoundTwoMember represents one member in a distributed key generating
+// group performing the second round of the TSS keygen.
+type tssRoundTwoMember struct {
+	*tssRoundOneMember
+}
+
+// MarkInactiveMembers takes all messages from the previous DKG protocol
+// execution phase and marks all member who did not send a message as IA.
```

Comment:
I think it's worth expanding "IA" to "Inactive" in the docstrings

Reply:
See d6bbdac0